ansible/roles/operator-pipeline/tasks/main.yml

```diff
@@ -50,38 +50,24 @@
         operator-pipeline.key: "{{ lookup('file', operator_pipeline_private_key_local_path, rstrip=False) | b64encode }}"
         operator-pipeline.pem: "{{ lookup('file', operator_pipeline_private_cert_local_path, rstrip=False) | b64encode }}"
 
-- name: Grant registry access
-  block:
-    - name: Create Operator pipeline quay token secret
-      no_log: yes
-      k8s:
-        state: present
-        force: yes
-        namespace: "{{ oc_namespace }}"
-        definition:
-          apiVersion: v1
-          kind: Secret
-          type: kubernetes.io/dockerconfigjson
-          metadata:
-            name: registry-dockerconfig-secret
-            labels:
-              app: operator-pipeline
-              suffix: "{{ suffix }}"
-              env: "{{ env }}"
-          data:
-            .dockerconfigjson: "{{ lookup('file', operator_pipeline_quay_token, rstrip=False) | b64encode }}"
-    - name: Link secret with pipeline service account (for imagestream operations)
-      k8s:
-        state: present
-        force: yes
-        namespace: "{{ oc_namespace }}"
-        definition:
-          apiVersion: v1
-          kind: ServiceAccount
-          metadata:
-            name: pipeline
-          secrets:
-            - name: registry-dockerconfig-secret
+- name: Create Operator pipeline quay token secret
+  no_log: yes
+  k8s:
+    state: present
+    force: yes
+    namespace: "{{ oc_namespace }}"
+    definition:
+      apiVersion: v1
+      kind: Secret
+      type: kubernetes.io/dockerconfigjson
+      metadata:
+        name: registry-dockerconfig-secret
```

Comment:
Is this supposed to be the same secret as the one from the instructions in `docs/rhc4tp-cluster.md`? I ask because that's not a secret for quay but rather the registry cluster. Do we need to use different names to avoid confusion or is there more to this?

Reply:
I feel I'm missing smething- that's a Quay token. It's used to push the bundle and index images to quay, and later to retrieve them from quay in rhc4tp cluster.